holoviews/element/path.py

```diff
@@ -148,7 +149,11 @@ def clone(self, *args, **overrides):
         settings = dict(self.get_param_values(), **overrides)
         if not args:
             settings['plot_id'] = self._plot_id
-        return self.__class__(*args, **settings)
+
+        pos_args = getattr(self, '_' + type(self).__name__ + '__pos_params', [])
```

Comment:
I partially remember - ``clone`` has to use ``getattr`` as not all elements need the positional parameters declared (most don't). Calling ``getattr`` will create a path on ``AttrTrees`` resulting in an annoying entry which doesn't happen for double underscored entries. You have to be careful with ``getattr`` when ``AttrTrees`` are involved...